charts/alertmanager/README.md

```diff
@@ -10,14 +10,14 @@ As per [prometheus.io documentation](https://prometheus.io/docs/alerting/latest/
 
 Kubernetes 1.14+
 
-## Get Repo Info
+## Get Repository Info
 
 ```console
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 helm repo update
 ```
 
-_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+_See [helm repository](https://helm.sh/docs/helm/helm_repo/) for command documentation._
```

Comment:
There is no 'helm repositiry' command. Please keep as it is. See Kube-prometheus-stack chart for example how to fix language issues correctly.

Reply:
```suggestion
_See [`helm repo`](https://helm.sh/docs/helm/helm_repo/) for command documentation._
```